spec/models/card_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+describe Card do
+  it 'should compare two words and add error if they equal' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.